Formula/apibuilder-cli.rb

```diff
@@ -0,0 +1,24 @@
+class ApibuilderCli < Formula
+  desc "Command-line interface to generate clients for api builder"
+  homepage "https://www.apibuilder.io"
+  url "https://github.com/apicollective/apibuilder-cli/archive/0.1.7.tar.gz"
+  sha256 "69a828ebb11270fbc507042d140c54a46ba8050b723fc75f55104f78abc13be9"
+
+  def install
+    system "./install.sh", prefix
+  end
+
+  test do
+    touch "config"
+    s = <<-EOS.undent
+    [default]
+    token = abcd1234
+   EOS
+    open("config", "w") do |f|
+      f.puts s
+    end
```

Comment:
This can be simplified to:
```
test do
  (testpath/"config").write <<-EOS.undent
    [default]
    token = abcd1234
  EOS
```

Reply:
thx -- fixed that, and squashed my commits